content/en/_index.html

```diff
@@ -3,26 +3,31 @@
 +++
```

Comment:
What is the default mode for the website or it auto-detects the browser settings ?

Reply:
It follows the computers dark/light mode by default, or uses light if it cant detect.